Translators/Incident/sample.env

```diff
@@ -0,0 +1,10 @@
+dual_carriageway_endpoint=https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded

```

Comment:
(non-blocking) What does "dual carriageway" refer to?

Reply:
That's the routes_withDEC portion - CDOT has two systems for their roadway and only dual_carriageway includes both ascending and descending directions.